README.md

```diff
@@ -105,3 +103,15 @@ enough tests pass to qualify the new operating system, then either merge the pul
 the pull request with bug fixes. If they just want to retrigger the tests due to potential test flakiness, they should
 push a new commit to do so, which would include a change to publish_and_test_config.yaml with the field
 'publish_dcos_images_after: never' to avoid rebuilding identical new images.
+
+## Running Locally
+
+This code can be executed locally by following theose steps:
+
+1) Add a your ssh public key to your AWS account
```

Comment:
Why is this required?

Reply:
Otherwise the old terraform installer was not working for me.

Reply:
When running locally, you can configure terraform to use your own ssh-key rather than the default dcos-images ssh-key that is used in the Jenkins automation.

Reply:
Shouldn't Terraform add the public key to AWS then?